src/AngleSharp.Diffing/Strategies/AttributeStrategies/DiffingStrategyPipelineBuilderExtensions.cs

```diff
@@ -63,5 +63,14 @@ public static IDiffingStrategyCollection AddStyleAttributeComparer(this IDiffing
             builder.AddComparer(StyleAttributeComparer.Compare, StrategyType.Specialized);
             return builder;
         }
+
+        /// <summary>
+        /// Enables the special style attributes comparer during diffing and orderers the styles alphabetically before comparing them.
+        /// </summary>
```

Comment:
I think we need to add a `<remarks>` section to both the extensions methods here explicitly stating that users should just use one of these, not both at the same time, as they will potentially conflict.

Reply:
I can just add a flag to the extension method for that? And default it to false?